addons/mail/models/mail_thread.py

```diff
@@ -2293,6 +2295,8 @@ def _notify_thread_by_email(self, message, recipients_data, msg_vals=False,
         for recipients_group_data in recipients_groups_data:
             # generate notification email content
             recipients_ids = recipients_group_data.pop('recipients')
+            if force_subtitles:
```

Comment:
no need to do this in the loop I think, you can do it just after
```
template_values = self._notify_by_email_prepare_rendering_contex
```

Reply:
Yes indeed, you are right